modules/sdk-coin-atom/test/unit/transaction.ts

```diff
@@ -0,0 +1,44 @@
+import { coins } from '@bitgo/statics';
+import should from 'should';
+import { Transaction } from '../../src';
+import * as testData from '../resources/atom';
+
+describe('Atom Transaction', () => {
+  let tx: Transaction;
+  const config = coins.get('tatom');
+
+  beforeEach(() => {
+    tx = new Transaction(config);
+  });
+
+  describe('Empty transaction', () => {
+    it('should throw empty transaction', function () {
+      should.throws(() => tx.toJson(), 'Empty transaction');
+      should.throws(() => tx.toBroadcastFormat(), 'Empty transaction');
+    });
+  });
+
+  describe('From raw transaction', () => {
+    it('should build a transfer from raw signed base64', function () {
+      tx.fromRawTransaction(testData.TEST_TX.signedTxBase64);
+      const json = tx.toJson();
+      should.equal(json.signerAddress, testData.TEST_TX.sender);
+    });
+    // it('should build a transfer from raw unsigned base64', function () {
```

Comment:
Will be adding more testing in a follow up PR, right now getting the skeleton's out to get closer to unblocking wallet creation via account lib in WP

Reply:
Can you add ticket numbers for this follow ups